Makefile

```diff
@@ -53,17 +53,18 @@ test-integration-cov:
 	go test ./tests/integration/... -timeout 30m -coverpkg=./... -coverprofile=integration-profile.out -covermode=atomic
 
 # run interchain tests
-# we can use PROVIDER_IMAGE_TAG, PROVIDER_IMAGE_NAME, SOUVEREIGN_IMAGE_TAG, and SOUVEREIGN_IMAGE_NAME to run tests with desired docker images,
-# including locally built ones that, for example, contain some of our changes that are not yet on the main branch.
-# if not provided, default value for PROVIDER_IMAGE_TAG and SOUVEREIGN_IMAGE_TAG is "latest" and for PROVIDER_IMAGE_NAME 
-# and SOUVEREIGN_IMAGE_NAME is "ghcr.io/cosmos/interchain-security"
+# we can use PROVIDER_IMAGE_TAG, PROVIDER_IMAGE_NAME, CONSUMER_IMAGE_TAG, CONSUMER_IMAGE_NAME, SOUVEREIGN_IMAGE_TAG, and SOUVEREIGN_IMAGE_NAME to run 
+# tests with desired docker images, including locally built ones that, for example, contain some of our changes that are not yet on the main branch.
+# if not provided, default value for image tag is "latest" and for image name is "ghcr.io/cosmos/interchain-security"
 test-interchain:
 	cd tests/interchain && \
 	PROVIDER_IMAGE_NAME=$(PROVIDER_IMAGE_NAME) \
 	PROVIDER_IMAGE_TAG=$(PROVIDER_IMAGE_TAG) \
 	SOUVEREIGN_IMAGE_NAME=$(SOUVEREIGN_IMAGE_NAME) \
```

Comment:
nit: "Sovereign" instead of "souvereign". Consider changing all instances

Reply:
fixed [b5faef4](https://github.com/cosmos/interchain-security/pull/2456/commits/b5faef4bed776a93b49973e526a8feb480f1fb61)